pom.xml

```diff
@@ -317,6 +317,7 @@
 					<failsOnError>true</failsOnError>
 					<failOnViolation>true</failOnViolation>
 					<configLocation>companieshouse_checks.xml</configLocation>
+					<violationIgnore>LineLength,CustomImportOrder</violationIgnore>
```

Comment:
We use intellij auto-format for both import sequence and line length (inherited limit of 100 is too little) - so we can ignore just those 2 for now